radicle-cli/src/commands/checkout.rs

```diff
@@ -19,11 +19,11 @@ pub const HELP: Help = Help {
     usage: r#"
 Usage
 
-    rad checkout <id> [<option>...]
+    rad checkout --node <nid> [<option>...]
 
 Options
 
-    --remote <id>   Remote namespace to checkout
+    --node <nid>    Remote namespace to checkout
```

Comment:
should the description here be changed from "Remote namespace to checkout"?

Reply:
No that's fine as-is